### ISSUE TITLE
🐛 issue with relative paths,

### ISSUE THREAD

**Issue by oviirup** [huozhi/bunchee#613]
I am working on a very simple project, code for the project will be given below. The issue with any code that is generated by bunchee, if there is a relative path, `./` is replaced by `.\\`.

### Reproduce issue

Create any project that rely on relative imports

```ts
// src/index.ts
export { slugify } from './slugify';
```

```ts
// src/slugify.ts
export function slugify(string: string) {
  return string
    .trim()
    .replace(/([a-z])([A-Z])/g, '$1-$2')
    .replace(/\W/g, (m) => (/[À-ž]/.test(m) ? m : '-'))
    .replace(/^-+|-+$/g, '')
    .replace(/-{2,}/g, '-')
    .toLowerCase();
}
```

Reference the files in `package.json` exports

```json
{
  "scripts": {
    "build": "bunchee"
  },
  "exports": {
    ".": {
      "default": "./dist/index.js",
      "types": "./dist/index.d.ts",
      "require": "./dist/index.cjs"
    },
    "./slugify": {
      "default": "./dist/slugify.js",
      "types": "./dist/slugify.d.ts",
      "require": "./dist/slugify.cjs"
    }
  }
}
```

Run build command

```bash
npm run build
```

### Current vs. Expected behavior

```js
// CURRENT 🡢 dist/index.js

export { slugify } from '.\\slugify.js';
                         ~~~~~~~~~~~~~~
```

```js
// EXPECTED 🡢 dist/index.js

export { slugify } from './slugify.js';
```

### Environment info

**OS:** Windows 11
**Package manger:** bun, yarn, pnpm

**package.json**

```json
{
  "name": "bunche-test",
  "type": "module",
  "exports": {
    ".": {
      "default": "./dist/index.js",
      "types": "./dist/index.d.ts",
      "require": "./dist/index.cjs"
    },
    "./nanoid": {
      "default": "./dist/nanoid.js",
      "types": "./dist/nanoid.d.ts",
      "require": "./dist/nanoid.cjs"
    },
    "./slugify": {
      "default": "./dist/slugify.js",
      "types": "./dist/slugify.d.ts",
      "require": "./dist/slugify.cjs"
    }
  },
  "files": ["dist"],
  "scripts": {
    "build": "bunchee"
  },
  "devDependencies": {
    "@types/node": "~22.10.1",
    "bunchee": "~6.0.2",
    "typescript": "~5.7.2"
  }
}
```

**Comment by oviirup** [comment]
The relative import issue also happens on `bunchee prepare`. It creates export paths with `.\\` as shown below.

```jsonc
"exports": {
  ".\\slugify": {
    "import": {
      "types": ".\\dist/es/.\\slugify.d.ts",
      "default": ".\\dist/es/.\\slugify.js"
    },
    "require": {
      "types": ".\\dist/cjs/.\\slugify.d.cts",
      "default": ".\\dist/cjs/.\\slugify.cjs"
    }
  },
}
// ...
```

**Comment by huozhi** [comment]
Thanks for reporting, fixed in 6.0.3
